pkgs/development/tools/revive/default.nix

```diff
@@ -0,0 +1,49 @@
+{ buildGoModule, fetchFromGitHub, lib }:
+
+buildGoModule rec {
+  pname = "revive";
+  version = "1.2.1";
+
+  src = fetchFromGitHub {
+    owner = "mgechev";
+    repo = "revive";
+    rev = "v${version}";
+    sha256 = "sha256-V0YWp4zQduW+cnuiVoEAPxDnsA4OR0TnJdoOF9TdRjw=";
+  };
+
+  ldflags = [
+    "-X"
+    "github.com/mgechev/revive/cli.version=${version}"
+    "-X"
+    "github.com/mgechev/revive/cli.date=2022-06-22"
+    "-X"
+    "github.com/mgechev/revive/cli.commit=23828cc39d4f91b69247bdea0ab4903bd6c1912e"
+    "-X"
+    "github.com/mgechev/revive/cli.builtBy=nixpkgs"
+  ];
+
+  # The following tests fail when built by nixpkgs:
+  #
+  # $ nix log /nix/store/build-revive.1.2.1.drv | grep FAIL
+  #
+  # --- FAIL: TestAll (0.01s)
+  #     --- FAIL: TestAll/context-keys-type.go (0.00s)
+  #     --- FAIL: TestAll/errorf.go (0.00s)
+  #     --- FAIL: TestAll/time-naming.go (0.00s)
+  #     --- FAIL: TestAll/var-declaration.go (0.00s)
+  # --- FAIL: TestTimeEqual (0.00s)
+  # --- FAIL: TestTimeNaming (0.00s)
+  # --- FAIL: TestUnhandledError (0.00s)
+  # --- FAIL: TestUnhandledErrorWithBlacklist (0.00s)
+  #
+  doCheck = false;
+
+  vendorSha256 = "sha256-Fpl5i+qMvJ/CDh8X0gps9C/BxF7/Uvln+3DpVOXE0WQ=";
```

Comment:
Can this be moved up just after `src` please

Reply:
👍